Lib9c/Policy/MaxBlockBytesPolicy.cs

```diff
@@ -58,7 +58,7 @@ private MaxBlockBytesPolicy(int defaultValue)
                     startIndex: 2_000_001,
                     value: 1024 * 1024 * 10))    // 10 MiB
                 .Add(new SpannedSubPolicy<int>(
-                    startIndex: 2_680_001,
+                    startIndex: 3_000_001,
```

Comment:
Since the internal cluster is isolated from the mainnet from the point of the fork, it is safe to hold it tight.

```suggestion
                    startIndex: 2_730_001,
```